bears/xml/XMLBear.py

```diff
@@ -22,11 +24,15 @@ class XMLBear(LocalBear, Lint):
     diff_message = "XML can be formatted better."
     output_regex = r'(.*\.xml):(?P<line>\d+): (?P<message>.*)\n.*\n.*'
     gives_corrected = True
+    use_stderr = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stdout_output:
+            # Return issues from stderr and stdout if stdout is not empty
+            return itertools.chain(
+                self._process_issues(self.stderr_output, filename),
+                self._process_corrected(self.stdout_output, filename, file))
```

Comment:
@Adrianzatreanu I have not added anything, the commits in this PR are unchanged cherry picks, they have already been acked